Unity3D/Assets/RosSharp/Scripts/TransformManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using System.Collections;
```

Comment:
Please rename the variables as follows:
linear_velocity -> linearVelocity
angular_velocity -> angularVelocity